assets/xml/objects/object_kin2_obj.xml

```diff
@@ -1,25 +1,25 @@
 ﻿<Root>
     <File Name="object_kin2_obj" Segment="6">
-        <DList Name="object_kin2_obj_DL_000120" Offset="0x120" />
+        <DList Name="gOceanSpiderHouseEmpty1DL" Offset="0x120" />
         <DList Name="gOceanSpiderHouseBombableWallDebrisDL" Offset="0x128" />
         <DList Name="gOceanSpiderHouseBombableWallCrackDL" Offset="0x2C0" />
         <DList Name="gOceanSpiderHouseBombableWallDL" Offset="0x360" />
         <Collision Name="gOceanSpiderHouseBombableWallCol" Offset="0x490" />
-        <DList Name="object_kin2_obj_DL_000650" Offset="0x650" />
-        <DList Name="object_kin2_obj_DL_000658" Offset="0x658" />
-        <Collision Name="object_kin2_obj_Colheader_000798" Offset="0x798" />
-        <DList Name="object_kin2_obj_DL_000820" Offset="0x820" />
+        <DList Name="gOceanSpiderHouseEmpty2DL" Offset="0x650" />
+        <DList Name="gOceanSpiderHouseSkullkidPaintingDL" Offset="0x658" />
+        <Collision Name="gOceanSpiderHouseSkullkidPaintingCol" Offset="0x798" />
+        <DList Name="gOceanSpiderHouseEmpty3DL" Offset="0x820" />
         <DList Name="gOceanSpiderHouseFireplaceGrateDL" Offset="0x828" />
-        <Collision Name="object_kin2_obj_Colheader_000908" Offset="0x908" />
-        <DList Name="object_kin2_obj_DL_000CA0" Offset="0xCA0" />
+        <Collision Name="gOceanSpiderHouseFireplaceGrateCol" Offset="0x908" />
+        <DList Name="gOceanSpiderHouseEmpty4DL" Offset="0xCA0" />
         <DList Name="gOceanSpiderHouseBookshelfDL" Offset="0xCA8" />
         <Collision Name="gOceanSpiderHouseBookshelfCol" Offset="0xF80" />
-        <DList Name="object_kin2_obj_DL_0010F0" Offset="0x10F0" />
+        <DList Name="gOceanSpiderHouseEmpty5DL" Offset="0x10F0" />
         <DList Name="gOceanSpiderHouseChestOfDrawersDL" Offset="0x10F8" />
         <Collision Name="gOceanSpiderHouseChestOfDrawersCol" Offset="0x1328" />
-        <!-- <Blob Name="object_kin2_obj_Blob_001354" Size="0x2C" Offset="0x1354" /> -->
+        <Texture Name="gOceanSpiderHouseUnusedTLUT1" Size="0x20" Offset="0x1360" />
```

Comment:
We'd like to move the `1` to be *before* the `TLUT`, similar to the DisplayLists above. Also, probably want an `OutName` for this?
```suggestion
        <Texture Name="gOceanSpiderHouseUnused1TLUT" OutName="ocean_spider_house_unused_tlut_1" Size="0x20" Offset="0x1360" />
```

Reply:
Got a problem with the map after changing this file. Will investigate before applying the changes in the object's XML file.